cli/hzn.go

```diff
@@ -735,6 +740,10 @@ Environment Variables:
 
 		// use HZN_EXCHANGE_NODE_AUTH for -n and trim the org
 		nodeIdTok = cliutils.WithDefaultEnvVar(nodeIdTok, "HZN_EXCHANGE_NODE_AUTH")
+
+		if err := version.VerifyExchangeVersion1(exchange.LoadExchangeVersion(*org, *userPw), false, true); err != nil {
```

Comment:
Here is opposite, -n is usually specified the node name and token to be created with -u cred. But it could also be used as a cred if the node already exists. So the logic should be: use userPw if it is not empty, else use nodeIdTok if it is not empty. If both are empty, just skip the version check.